pkgs/by-name/at/atproto-goat/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+buildGoModule {
+  pname = "atproto-goat";
+  version = "0-unstable-2024-10-08";
+
+  src = fetchFromGitHub {
+    owner = "bluesky-social";
+    repo = "indigo";
+    rev = "06bacb465af714feb77609566aba15ab1ed41e24";
+    # To get proper version information
+    leaveDotGit = true;
```

Comment:
You may want to patch the version script instead. `leaveDotGit` is sometimes unreproducible thus isn't preferred